tests/fixtures.py

```diff
@@ -758,6 +760,33 @@ def parametrize(params, values, **kwargs):
             True,
         ),
         ("_1c444", "function=0x1F68, bb=0x1F68, insn=0x1FF9", capa.features.insn.API("FromHbitmap"), False),
+        ("_1c444", "token=0x600002B", capa.features.insn.Property("System.IO.FileInfo::Length"), True),
+        ("_1c444", "token=0x6000081", capa.features.insn.API("System.Diagnostics.Process::Start"), True),
+        (
+            "_1c444",
+            "token=0x6000081",
+            capa.features.insn.Property("System.Diagnostics.ProcessStartInfo::UseShellExecute"),
+            True,
+        ),
+        (
+            "_1c444",
+            "token=0x6000081",
+            capa.features.insn.Property("System.Diagnostics.ProcessStartInfo::WorkingDirectory"),
+            True,
+        ),
+        (
+            "_1c444",
+            "token=0x6000081",
+            capa.features.insn.Property("System.Diagnostics.ProcessStartInfo::FileName"),
+            True,
+        ),
+        ("_0953c", "token=0x6000004", capa.features.insn.Property("System.Diagnostics.Debugger::IsAttached"), True),
+        (
+            "_692f",
+            "token=0x6000006",
+            capa.features.insn.Property("System.Management.Automation.PowerShell::Streams"),
+            False,
+        ),
```

Comment:
This is great, thank you! I believe we are emitting `Property` features using 3 different techniques in .NET (via MethodDef, MemberRef, and field instructions). Do we have all three methods covered here? If not, are we able to find examples of each in our existing test binaries?

Reply:
Added new tests to cover all the methods.